dj_rql/filter_cls.py

```diff
@@ -243,6 +243,12 @@ def build_q_for_filter(self, data):
         """
         filter_name, operator, str_value = data.filter_name, data.operator, data.str_value
         list_operator = data.list_operator
+        filter_basename, namespace = data.filter_basename, data.namespace
+
+        if namespace and filter_basename == RQL_SEARCH_PARAM:
+            raise RQLFilterLookupError(details={
+                'error': f'Filter "{filter_basename}" can be applied only on a global level.',
```

Comment:
If we introduce this kind of message, I would suggest reusing it for all the stuff that follows the same patterns. Ping me to discuss that.

Reply:
Discussed. Grammar error fixed